site/src/App.tsx

```diff
@@ -25,17 +29,19 @@ export const AppProviders: FC<PropsWithChildren> = ({ children }) => {
   return (
     <HelmetProvider>
       <StyledEngineProvider injectFirst>
-        <ThemeProvider theme={dark}>
-          <CssBaseline enableColorScheme />
-          <ErrorBoundary>
-            <QueryClientProvider client={queryClient}>
-              <AuthProvider>
-                {children}
-                <GlobalSnackbar />
-              </AuthProvider>
-            </QueryClientProvider>
-          </ErrorBoundary>
-        </ThemeProvider>
+        <MuiThemeProvider theme={dark}>
+          <EmotionThemeProvider theme={dark}>
```

Comment:
Do we need both? Or is it only for the transition?

Reply:
Well, we can't get rid of `MuiThemeProvider` as long as we use MUI, and making Emotion aware of the theme has some nice DX upsides, so this is gonna persist for a while.